src/pages/library/Showcase.js

```diff
@@ -16,104 +18,337 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
+// Description of the structure of the sidebar. Case is preserved when rendering but all matching is case-insensitive.
+const sidebarSections = [
+  {
+    name: 'Getting Started',
+    labels: ['Workflow Tutorials', 'Notebook Tutorials', 'Data Tutorials', 'RStudio Tutorials', 'Galaxy Tutorials'],
+    keepCollapsed: true
+  },
+  {
```

Comment:
Would this increase concision without harming readability?
```suggestion
  }, {
```

Reply:
Would you then put the first open and last close on the same lines as the opes and close square brackets as well? Feels unbalanced otherwise, but I'm not sure I like the end result:
```
const sidebarSections = [{
  name: 'Getting Started',
  labels: ['Workflow Tutorials', 'Notebook Tutorials', 'Data Tutorials', 'RStudio Tutorials', 'Galaxy Tutorials'],
  keepCollapsed: true
}, {
  ...
}, {
  name: 'Projects',
  labels: ['HCA', 'AnVIL', 'BRAIN Initiative', 'BioData Catalyst', 'NCI']
}]

Reply:
Yes. This is our style, for better or worse. I think it's pretty readable once you get used to it.